### PR TITLE
Added span request name option to echo tracer

### PR DESCRIPTION
**Pull request by aneshas**
Adding a span name option to echo apm middleware such as the one apmhttp has

### CODE REVIEW

File: module/apmechov4/middleware.go [hunk 1]
```diff
@@ -167,6 +178,18 @@ func WithTracer(t *apm.Tracer) Option {
 	}
 }
 
+// WithRequestName returns an Option which sets r as the function
+// to use to obtain the span name for the given echo request.
```
Comment (axw) [inline module/apmechov4/middleware.go:182]:
```suggestion
// to use to obtain the transaction name for the given echo request.
```

**Review by axw — APPROVED** [review]
Thanks @aneshas! Looks good, just one small nitpick about a doc comment.

File: module/apmechov4/middleware.go (hunk 1 above)
Comment (aneshas) [reply module/apmechov4/middleware.go:182]: Yes makes sense

**Comment by axw** [comment]
Thanks :)

**Comment by aneshas** [comment]
@axw might be a stupid question, but how does this actually get released ?

**Comment by axw** [comment]
@aneshas not a stupid question! We generally create new releases around the time of Elastic Stack minor releases (next one being 8.4.0). That's expected within weeks, so if you would like to update sooner, I would suggest using the unreleased commit (update modules to `@latest`) in your go.mod.

**Comment by aneshas** [comment]
Ah tnx

**Pull request merged**